### ISSUE TITLE
User connection notifications

### ISSUE THREAD

**Issue by demiurge28** [Ylianst/MeshCentral#5972]
Ability to have "Slack" for my use, notifications of User connections/disconnections to specific designated systems or group. 
Have it in the window you would select at a group or system level to message notifications.
![Mesh Notifications](https://github.com/Ylianst/MeshCentral/assets/3865905/4cbe16c6-bccc-47a8-b7f6-9834259060c2)

Info that is sent would be time/date, user, system, and event.

**Comment by si458** [comment]
slack was added as a messaging notification method already here https://github.com/Ylianst/MeshCentral/issues/5564
enable it here - https://ylianst.github.io/MeshCentral/messaging/#slack-setup
then setup a webhook and add the webhook into your user messaging notification

**Comment by demiurge28** [comment]
I know slack was added as I requested it. What I wanted was the ability for mesh to notify when users connected/disconnected from the system with pertinent info in regards to the user, like how I can get notified with system connect/disconnect.

**Comment by si458** [comment]
@demiurge28 ah yes sorry! mis-read the request
the is already a few open with this in mind #3460 #3715
but yes i do agree, while the mesh keeps a log in its events of connect and disconnects to devices,
the is no notifications via messaging or emails etc at the moment as far aw im aware!

**Comment by demiurge28** [comment]
Yes I see via the other request you linked to, maybe in the future. Thank you for responding promptly.

**Comment by Mfinadmin** [comment]
Is this feature available now ?
